velox/connectors/hive/storage_adapters/s3fs/S3FileSystem.cpp

```diff
@@ -640,6 +646,52 @@ class S3FileSystem::Impl {
     return getDefaultCredentialsProvider();
   }
 
+  // Return a client RetryStrategy based on the config.
+  std::optional<std::shared_ptr<Aws::Client::RetryStrategy>> getRetryStrategy()
+      const {
+    auto retryMode = hiveConfig_->s3RetryMode();
+    auto maxAttempts = hiveConfig_->s3MaxAttempts();
+    if (retryMode.has_value()) {
+      if (retryMode.value() == "standard") {
+        if (maxAttempts.has_value()) {
+          VELOX_USER_CHECK(
+              (maxAttempts.value() > 0),
+              "Invalid configuration: specify 'hive.s3.max-attempts' > 0.");
```

Comment:
Maybe print the value?
Should 0 be allowed here?
`VELOX_USER_CHECK(
              (maxAttempts.value() >= 0), "Invalid configuration: specified 'hive.s3.max-attempts' value {} is < 0.", maxAttempts.value());`

Reply:
As checked in [CilentConfiguration](https://github.com/aws/aws-sdk-cpp/blob/main/src/aws-cpp-sdk-core/source/client/ClientConfiguration.cpp#L394), it can be `0`. Changed it to use `VELOX_USER_CHECK_GE`. @jinchengchenghh